internal/impl/devbox.go

```diff
@@ -58,6 +58,16 @@ func InitConfig(dir string, writer io.Writer) (created bool, err error) {
 			color.HiYellowString(s),
 		)
 	}
+	// .envrc file creation
+	envrcfilePath := filepath.Join(dir, ".envrc")
+	filesExist := plansdk.FileExists(envrcfilePath)
+	if !filesExist { // don't overwrite an existing .envrc
+		err := generate.CreateEnvrc(tmplFS, dir)
+		if err != nil {
+			return false, errors.WithStack(err)
```

Comment:
Should we ignore this? if `.envrc` creation fails, should we just move forward with devbox init flow and not interrupt?

Reply:
yeah, you can print a warning via `debug.Log` but probably not worth concerning a user about.